src/content/admin/cloud-credentials/gcp.mdx

```diff
@@ -20,10 +20,20 @@ GCP credentials are configured using [Workload Identity Federation](https://clou
 
 As a first step let's create a Workload Identity Pool. A Workload Identity Pool is a group of identities that can be used to access GCP resources.
 
+Okteto displays the OIDC endpoint of your cluster in the Admin -> General view of the Dashboard:
```

Comment:
I would present it this way: first showing where to obtain the data and then how to use it.